README.md

```diff
@@ -353,14 +343,11 @@ For example:
 
 ```json
 {
-    "gamepad" : {
-        "id" : "motion-controller-id",
-        "handedness" : {
-            "none" : {
-                "asset" : "some-url",
-                "root" : "none-handedness-controller",
-                "components" : [0]
-            }
+    "handedness" : {
+        "none" : {
+            "asset" : "some-url",
+            "root" : "neutral-handedness-controller",
```

Comment:
This looks like it was changed to "none-handedness-controller" below. Applicable here as well, I assume?